test/unit/css_consolidation_test.rb

```
@@ -0,0 +1,115 @@
+    # This test validates button styling after component extraction
+  end
+
+  def test_homepage_responsive_utility_baseline
+    doc = parse_html_file("index.html")
+
+    # Check for responsive classes and viewport meta
+    viewport_meta = doc.css("head meta[name='viewport']").first
+    refute_nil viewport_meta, "Homepage must have responsive viewport meta tag"
+
+    # FAILING TEST: This will fail until responsive utilities are extracted
+    responsive_classes = doc.css("[class*='mobile'], [class*='tablet'], [class*='desktop']")
```

RuboCop (1.80.2)

[warning] Useless assignment to variable - `responsive_classes`.

(Lint/UselessAssignment)